src/app/legacy/containers/MostRead/Canonical/Item/__snapshots__/index.test.jsx.snap

```diff
@@ -161,8 +161,8 @@ exports[`MostReadItemWrapper should render ltr correctly with 10 items 1`] = `
 }
 
 .emotion-7 {
-  font-family: ReithSerif,Helvetica,Arial,sans-serif;
-  font-weight: 300;
+  font-family: Helmet,Freesans,Helvetica,Arial,sans-serif;
```

Comment:
We had to tidy up some of the legacy tests to ensure that the new tests would also work - the legacy snapshots were updated, and then copied to the new component for consistency